api/python/provisioner/commands/auto_deploy.py

```diff
@@ -77,9 +96,19 @@ def run(self, nodes, **kwargs):
                 ), targets=setup_ctx.run_args.primary.minion_id
             )
 
+        logger.info("Deployment Pre-Flight Validations")
+
+        self.deployment_validations(GroupChecks.PRE_CHECKS.value)
```

Comment:
why we use something named `PRE_CHECKS`: is it deployment specific or makes sense to check for any routine (swupdate, unboxing)

Reply:
Initially, when started out with Deployment checks, we named them pre_checks and post_checks. 
Now updated. Comment addressed.